tests/micropython/run_testcase.py

```diff
@@ -0,0 +1,121 @@
+#! ../../../../../../lv_micropython/ports/unix/build-standard/micropython
```

Comment:
I think we could delete this line, the path will be different for every developer

Reply:
As LVGL is a submodule in `lv_binding_micropython` it has a fixed place and this path shouldn't really change. Or am I missing something?

Reply:
If we run the tests from `lv_micropython`, then the path can be relative, but 1 level shorter:
`../../../../../ports/unix/build-standard/micropython`     (without `../lv_micropython/` )

Reply:
True!